find the smurf/script.js

```diff
@@ -0,0 +1,56 @@
+let img = document.querySelector(".hide");
+let game = document.querySelectorAll(".game");
+let modal = document.querySelector(".quiz-over-modal");
+let btnTryAgain = document.getElementById("btn-try-again");
+let modalTwo = document.querySelector(".modal");
+let btnTryAgainTwo = document.getElementById("btn-tr-again");
+alert("Привет!, нажми на смурфика и он спрячется!");
+img.addEventListener("click", () => {
+  computer();
+  alert("Найди его!");
+  var audio = new Audio(); 
+  audio.src = "The Minions.mp3"; 
+  audio.autoplay = true;
+});
+function computer() {
+  computerChoice = Math.floor(Math.random() * game.length);
+  console.log(computerChoice);
+  if (computerChoice == 0) {
+    computerChoice = 3;
+  }
+  if (computerChoice === 1) {
+    img.style.display = "block";

```

Comment:
[DRY](https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself)

You should avoid copy-paste